src/FiboApp.java

```diff
@@ -0,0 +1,5 @@
+public class FiboApp {
```

Comment:
Please rename this file to a meaningful name. FiboApp is confusing. So change it to FibonnacciApp so that we exactly know what this app is for.

Reply:
Thats good suggestion. I will make this change quickly. Thanks